metadata/provider_metadata.json

```diff
@@ -17554,6 +17554,257 @@
                 }
             }
         ],
+        "ibm_dl_route_report": [
```

Comment:
This file is auto-generated during release.. Kindly remove changes from this file